src/mcp/tools/storage/get_download_url.ts

```diff
@@ -0,0 +1,29 @@
+import { z } from "zod";
+import { tool } from "../../tool.js";
+import { toContent } from "../../util.js";
+import { getDownloadUrl } from "../../../gcp/storage.js";
+
+export const get_storage_object_download_url = tool(
```

Comment:
nit: `get_object_download_url`.

It will become `storage_get_object_download_url` after this block.

https://github.com/firebase/firebase-tools/blob/master/src/mcp/tools/index.ts#L31

Reply:
Done!